### ISSUE TITLE
Select: `aria-expanded` is always on `false`, it should be true when the dropdown is opened.

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#1966]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

We should also check the if `limel-menu` behaves correctly.
